extension/src/popup/components/manageNetwork/NetworkForm/index.tsx

```diff
@@ -197,7 +191,7 @@ export const NetworkForm = ({ isEditing }: NetworkFormProps) => {
   };
 
   const supportsSorobanRpc = (network: string) =>
-    network === NETWORK_NAMES.FUTURENET;
+    network === NETWORK_NAMES.FUTURENET || network === NETWORK_NAMES.TESTNET;
```

Comment:
Oh, I think you can actually use the selector `settingsSorobanSupportedSelector` in the `settings` duck for this

Reply:
ah thanks, done in 414609fb1ce5739d8b32cfa05652686e3ab81c87